README.md

```diff
@@ -69,108 +69,116 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.56 - February 2022 Update
+### 0.57 - March 2022 Update
 
 In this release, we focused heavily on stability and improvements.  Below are some of the highlights!
 
 **Highlights**
 
-- Upgrade notes!  A big feedback items for new versions was upgrade notes.  We have the release notes on GitHub as well.
-- FancyZones zone limit has been increased to 128 zones.  Before the limit was 40.
-- Timezone conversion plugin for PowerToys Run!  Thanks [@TobiasSekan](https://github.com/TobiasSekan)
-- Child and Popup window support for FancyZones.  To enable, go to Settings→FancyZones→Windows. These were bugs we fixed that without realizing it was a useful feature to most.
-- Find my mouse will now activate via shaking the mouse with a settings change!
-![Find my mouse setting for Activate to shake](https://user-images.githubusercontent.com/1462282/156048784-5a16ae0e-3551-47c6-a601-833acc9e893b.png)
+- PowerToys Run quality pass. Old standing issues were re-evaluated and fixed.
+- Additional features and improvements were added to existing PowerToys Run plugins.
+- New plugin for time and date values/information in PowerToys Run. Thanks [@htcfreek](https://github.com/htcfreek)!
+- The [PowerToys Run documentation](https://aka.ms/PowerToysOverview_PowerToysRun) is also receiving a required update. Thanks [@htcfreek](https://github.com/htcfreek)!
+- We're registering svgs as a picture kind when SVG Thumbnails are enabled so they appear when searching for pictures in File Explorer.
+- We've disabled PDF preview by default, given its incompatibilities with Outlook and that Edge is now being registered for previewing PDF files on Windows 10 too.
+- From a coding quality point of view, every project now has code analyzer active. Thanks [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper)!
+- A double click on the tray icon is needed instead of single click to open settings.
 
 ### Always on Top
 
-- Fixed excess GPU / CPU usage when enabled
-- If border has focus, not closable via F4
-- Changing border sizes should resize correctly for existing windows
-- Border goes away with Outlook modal windows 
+- New sound for Always on Top activation. Thanks [@franky920920](https://github.com/franky920920)!
```

Comment:
```suggestion
- Different sound for Always on Top activation. Thanks [@franky920920](https://github.com/franky920920)!
```

Reply:
"We changed the sound" makes most sense here.

Reply:
New gets that novelty feeling ;) It's a new sound being used, even if the sound is not new.